### ISSUE TITLE
ESlint output nothing, when I break the rule `order-in-components`

### ISSUE THREAD

**Issue by lakb248** [vuejs/eslint-plugin-vue#110]
**Tell us about your environment**

* **ESLint Version:** 4.3.0
* **eslint-plugin-vue Version:** 3.8.0
* **Node Version:** v6.11.0

**Please show your full configuration:**

```
{
  "extends": ["eslint:recommended", "plugin:vue/recommended"],
  "plugins": [
    "html"
  ],
  "parserOptions": {
    "parser": "babel-eslint",
    "sourceType": "module"
  },
  "rules": {
    "indent": [2, 4],
    "semi": [2, "always"],
    "no-negated-condition": 0,
    "no-lonely-if": 0,
    "eqeqeq": [2, "allow-null"],
    "no-eq-null": 0,
    "quote-props": [2, "as-needed"],
    "no-warning-comments": 0,
    "max-len": ["error", 120],
    "comma-dangle": ["error", "never"],
    "arrow-parens": ["error", "as-needed"],
    "no-invalid-this": 0
  }
}

```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
    <button class="wl-button"
        :class="[
            type ? 'wl-button--' + type : '',
            size ? 'wl-button--' + size : '',
            {
                'wl-button--plain': plain,
                'wl-button--disabled': disabled
            },
        ]"
        @click="onClick"><slot></slot></button>
</template>

<script>
    export default {
        methods: {
            onClick(event) {
                this.$emit('click', event);
            }
        },
        props: {
            type: {
                type: String
            },
            size: {
                type: String
            },
            disabled: Boolean,
            plain: Boolean
        },
        name: 'WlButton'
    };
</script>
```

**What did you expect to happen?**
Because I put the `name` property after `props`, so I hope it will throw error because [order-in-components](https://github.com/vuejs/eslint-plugin-vue/blob/master/docs/rules/order-in-components.md) rule, but the `eslint` output nothing 

How to config it correctly???

**Comment by armano2** [comment]
vue/order-in-components by default is disabled, try to add it to config.

_Sent from my Xiaomi Redmi 4A using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_

**Comment by armano2** [comment]
@lakb248  `vue/order-in-components` by default is disabled, try to add it to config.

**Comment by lakb248** [comment]
@armano2 Thanks a lot, I found that `plugin:vue/recommended` rules will be conflict with `eslint:recommend` rules when version of eslint is 3.19.0, because the eslint-recommended rules as below:
```javascript
...
module.exports = {
    parser: "espree",
    ecmaFeatures: {},

    rules: {

        /* eslint-enable sort-keys */
        "accessor-pairs": "off",
        "array-bracket-spacing": "off",
        "array-callback-return": "off",
        "arrow-body-style": "off",
        "arrow-parens": "off",
        "arrow-spacing": "off",
...
```
and the basic rules of `eslint-plugin-vue` is:
``` javascript
module.exports = {
  root: true,

  parser: require.resolve('vue-eslint-parser'),

  parserOptions: {
    ecmaVersion: 6,
    sourceType: 'module',
    ecmaFeatures: {
      jsx: true,
      experimentalObjectRestSpread: true
    }
 }
...
```
the parser config will be override.

**Comment by michalsnik** [comment]
Looks like it's solved :) Cool
